charts/jenkins/values.yaml

```
@@ -142,6 +142,10 @@ master:
   # When setting runAsUser to a different value than 0 also set fsGroup to the same value:
   runAsUser: 1000
   fsGroup: 1000
+  # If you have PodSecurityPolicies that require dropping of capabilities as suggested by CIS K8s benchmark, put them here
+  securityContextCapabilities: false
```

```suggestion
  securityContextCapabilities: {}
```

Using empty block `{}` might be better than using `false`.